submodules/eps/__init__.py

```diff
@@ -1,68 +1,140 @@
 import logging
 import time
 import smbus
+import yaml
 
 from smbus2 import SMBusWrapper
 from functools import partial
 
 from core.mode import Mode
 from core.threadhandler import ThreadHandler
+from core.error import Error
+from core.log import Log
+from submodules import telemetry
 
 # Initialize global variables
 logger = logging.getLogger("EPS")
 address = 0x57
```

Comment:
Do not initialize variables outside a method scope b/c these variables will be reinitialized whenever the module is imported. (i.e. variables will be reinitialized if two different files import this file). Instead, initialize these variables (and any other global variables) in the `start()` method so they are only initialized once.
```
def start():
    global epsdict
    epsdict = {...}
```